examples/RM3/wecSimInputFile.m

```diff
@@ -63,12 +64,16 @@
     % Body Mass. The 'equilibrium' Option Sets it to the Displaced Water 
     % Weight.
 body(1).inertia = [20907301 21306090.66 37085481.11];  % Moment of Inertia [kg*m^2]     
+body(1).largeXYDisplacement.option = 0;
```

Comment:
```suggestion
body(1).largeXYDisplacement.option = 0; % Change to 1 to account for large XY displacements
```

Reply:
Hi @jtgrasb 

Thanks for your review!

I didn't make a dedicated example for this yet: I only modified the existing RM3 example for the purposes of testing. On those grounds, I would prefer to leave the RM3 example file as it was, without any of my changes, since it is meant to be an approachable "first time user" sort of example. I should make an example file for WEC-Sim Applications, but I'll do that in a separate PR. 

Were you able to build and display the docs properly? I was having trouble on my local machine but that is more likely my own faulty Python environment than anything I did in the *rst.

Reply:
@dforbush2 

Makes sense on the RM3 example, just wasn't sure since it was in the PR. 

Yeah, docs built and looked fine!

Reply:
same comment. I'm reverting this for the merge, but it'll be on Dom's fork in a prior commit.